applications/Chat/benchmarks/benchmark_opt_lora_dummy.py

```diff
@@ -76,9 +76,9 @@ def main(args):
     if args.strategy == "ddp":
         strategy = DDPStrategy()
     elif args.strategy == "colossalai_gemini":
-        strategy = GeminiStrategy(placement_policy="cuda", initial_scale=2**5)
+        strategy = GeminiStrategy(placement_policy="auto", initial_scale=2**5)
```

Comment:
This is not equivalence. Bounding `"static"` to `"colossalai_gemini"` and `"auto"` to `"colossalai_gemini_auto"` is better.

Reply:
updated